loolwsd.xml.in

```diff
@@ -186,4 +186,38 @@
     <monitors desc="Addresses of servers we connect to on start for monitoring">
     </monitors>
 
+    <freemium desc="Freemium settings">
+        <disabled_commands desc="List of commands to disable the functionality. Each new command should be added to new line.">
```

Comment:
This contradicts the comment that any whitespace is expected!

Let's keep it simple and not force "styling" on users. Simple space should be fine; there is no reason why we have to force them to use new-lines (also, notice that the below is *not* just new-lines, there are spaces too, so the comment is yet again inaccurate because it implies strict new-lines are required when in fact simple spaces will work too, as it obviously has spaces! So the comment becomes a way to force "style" on users, which is a bit of an overreach, I think.)

Reply:
forcing a little styling and increasing the readability would eventually be helpful to the user, with a long list and just spaces it gets difficult to identify the mistake. I have handled the spaces too.

Reply:
This is a bad idea because it makes the code more complex and has the unwanted side-effect of annoying users without buying us anything of value (it already cost us something to make it more complex). It also introduces new special case ("this setting requires new-line") when all the other cases do not.

I understand that it's not fun to remove something after making an effort to build it, but it's even better to discuss similar user-facing decision before adding them, or at least remember that the purpose is to make the user's life easier (even if it means we do more work, or remove our favorite code).

At any rate, I don't insist. I just raised a number of issues with this unnecessary design decision, including the questionable addition of boost.

Reply:
boost is already removed. I realised it was a mistake boost wasn't in online dependency but rather in my system installed that's why it worked. I did not mean to introduce a new dependency obviously. 



> It also introduces new special case ("this setting requires new-line") when all the other cases do not.

This makes sense to me :)